cdap-ui/app/cdap/components/DataPrepConnections/KafkaConnection/index.js

```diff
@@ -248,9 +327,17 @@ export default class KafkaConnection extends Component {
 
   /** Return true if there is some error. */
   testInputs() {
-    let isSomeError = Object.keys(this.state.inputs).some(key => this.state.inputs[key]['error'] !== '');
+    let isSomeError = Object.keys(this.state.inputs).some(key => {
+      if (Array.isArray(this.state.inputs[key])) {
+        const isSomeErrInArr = this.state.inputs[key].reduce((a,b) => { return a.concat(b);  })
```

Comment:
Instead of a,b give logical name and assign initial value in reduce function (it might fail as a is undefined in first run)